tailwind.base.config.js

```diff
@@ -66,12 +66,6 @@ module.exports = {
           },
         ],
         28: '28px',
-        48: [
-          '48px',
-          {
-            lineHeight: '58px',
-          },
-        ],
```

Comment:
Wondering if we shouldn't prevent mixing the use of pre-generated and JIT classes for font sizes. Maybe we should set `text-13` and `text-21` via JIT as well if this is the preferred way!?

Reply:
yes, absolutely agree, these preset classes do not make sense now that we have JIT mode for tailwind. I didn't do it here since I feared the PR would get too big. I guess we should do this progressively with other PRs.